archey/entries/cpu.py

```diff
@@ -19,6 +19,8 @@ class CPU(Entry):
     Each `dict` **SHOULD** contain only one entry (CPU model name as key and cores count as value).
     """
 
+    _ICON = "\uf4bc"  # UTF-8 Code
```

Comment:
It might seem "picky" but do you think specifying Nerd font icon "pretty names"  instead of "UTF-8 Code" would be more relevant ?

Reply:
Something could be done, it increases the dependencies or my icon.py file is needed. To get around all this, I used the direct path.
I have manually dissolved everything and written directly into it.

This was the original reason why I added the file. 

https://pypi.org/project/nerdfonts/
```python
import nerdfonts as nf

print(nf.icons['oct_cpu']) # for printing one icon with the name oct_cpu
``` 
or

```python
import icons
print (icons.icons['oct_cpu']) # for printing one icon with the name oct_cpu

```